src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -3954,6 +3954,48 @@
+    ("SveExtractVectorTest.template",        new Dictionary<string, string> { ["TestName"] = "SveExtractVector_Int16_1",                                                                               ["Isa"] = "Sve",       ["LoadIsa"] = "Sve", ["Method"] = "ExtractVector",                                                     ["RetVectorType"] = "Vector", ["RetBaseType"] = "Int16",   ["Op1VectorType"] = "Vector", ["Op1BaseType"] = "Int16",  ["Op2VectorType"] = "Vector", ["Op2BaseType"] = "Int16",                                                               ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetInt16()",         ["NextValueOp2"] = "TestLibrary.Generator.GetInt16()",         ["ElementIndex"] = "1",                                                        ["ValidateIterResult"] = "Helpers.ExtractVector(firstOp, secondOp, ElementIndex, i) != result[i]"}),
+    ("SveExtractVectorTest.template",        new Dictionary<string, string> { ["TestName"] = "SveExtractVector_Int32_1",                                                                               ["Isa"] = "Sve",       ["LoadIsa"] = "Sve", ["Method"] = "ExtractVector",                                                     ["RetVectorType"] = "Vector", ["RetBaseType"] = "Int32",   ["Op1VectorType"] = "Vector", ["Op1BaseType"] = "Int32",  ["Op2VectorType"] = "Vector", ["Op2BaseType"] = "Int32",                                                               ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetInt32()",         ["NextValueOp2"] = "TestLibrary.Generator.GetInt32()",         ["ElementIndex"] = "1",                                                        ["ValidateIterResult"] = "Helpers.ExtractVector(firstOp, secondOp, ElementIndex, i) != result[i]"}),
+    ("SveExtractVectorTest.template",        new Dictionary<string, string> { ["TestName"] = "SveExtractVector_Int64_1",                                                                               ["Isa"] = "Sve",       ["LoadIsa"] = "Sve", ["Method"] = "ExtractVector",                                                     ["RetVectorType"] = "Vector", ["RetBaseType"] = "Int64",   ["Op1VectorType"] = "Vector", ["Op1BaseType"] = "Int64",  ["Op2VectorType"] = "Vector", ["Op2BaseType"] = "Int64",                                                               ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetInt64()",         ["NextValueOp2"] = "TestLibrary.Generator.GetInt64()",         ["ElementIndex"] = "1",                                                        ["ValidateIterResult"] = "Helpers.ExtractVector(firstOp, secondOp, ElementIndex, i) != result[i]"}),
```

Comment:
I didn't notice it earlier, but we had these test alphabetical for easy searching.

Reply:
I can fix the ordering.